website/docs/addon_usd_artist_maya_workflow.md

```diff
@@ -0,0 +1,140 @@
+
+![](assets/usd/ayon_usd_maya_products_after_publish.gif)
+![](assets/usd/ayon_usd_maya_products_after_publish_web.gif)
+
+The AYON asset contribution workflow automates the generation of the layers for you.
+
+Above we published a `usdMain` product, which had USD contributions enabled and targeted the product `usdAsset` with the department layer `model`.
+
+So it generated:
+
+- The target `usdAsset` product.
+- The department layer for the `usdAsset`, specifically `usdAsset_{layer}` -> `usdAsset_model`.
+- The `usdMain` product is added into that department layer, so the `usdMain` product is in the `usdAsset_model` layer.
+
+Generating the asset structure:
+```
+usdAsset (target product)
+  - usdAsset_model (department layer)
+    - usdMain (product of the instance)
+```
+
+If you were to publish multiple products at once, say we would generate `usdModelMain` and `usdModelDamaged`. It would then generate:
+
+```
+usdAsset (target product)
+  - usdAsset_model (department layer)
+    - usdModelMain
+    - usdModelDamaged
+```
+
+#### Contributions are additive
+
+Over time, contributing to the asset adds more products to the `usdAsset`, across likely a variety of department layers.
+Say our current publish structure is:
+```
+usdAsset (target product)
+  - usdAsset_model (department layer)
+    - usdModelMain (single contribution product)
+```
+
+Now publishing a `usdLookMain` to department layer `look` would make a new `usdAsset` version:
+```
+usdAsset (target product)
+  - usdAsset_model (department layer)
+    - usdModelMain (single contribution product)
+  - usdAsset_look (department layer)
+    - usdLookMain (single contribution product)
+```
+
+Then publishing another look variant `usdLookDamaged` would make a new `usdAsset` version:
+
+```
+usdAsset (target product)
+  - usdAsset_model (department layer)
+    - usdModelMain (single contribution product)
+  - usdAsset_look (department layer)
+    - usdLookMain (single contribution product)
+    - usdLookDamaged (single contribution product)
+```
+
+:::note Contribution workflow is 'additive' to the target product's department layers
+
+Whenever publishing a product with contribution enabled it will add it to an existing target product's department layers (or _update_ if it's the exact same product).
+
+This workflow makes it possible for different departments to simultaneously work on an asset and contribute to it additively.
+
+:::
+
+:::note Work in Progress - Removing contributions 
+
+There currently are no ready-to-go AYON tools that make it trivial to remove existing contributions from a product or department layer.
+
+Unfortunately this means if you currently accidentally make a wrong contribution you will have to go and edit the actual USDA files' contents to remove entries.
+
+_Warning: This can be a technical process!_
+
+Providing tools for [easy editing of these existing contributions is on our issue tracker](https://github.com/ynput/ayon-usd/issues/23).
+
+:::
```

Comment:
This section may be much better on the USD contribution page and have it link to that from this Maya page. @LiborBatek @MustafaJafar what do you think?

Reply:
Yeah, lets move it to the workflow page.

Reply:
I thought about it again after reading through the page. 
tbh, I expected explaining how the creator will modify maya's outliner.. I'm not sure if artists will worry about the end published results as much as they will worry about the outliner. 

Also, will that affect any connected attributes ?

Reply:
> Also, will that affect any connected attributes ?

No - should not.

> I'm not sure if artists will worry about the end published results as much as they will worry about the outliner.

I've added more details with this commit: https://github.com/ynput/ayon-documentation/pull/205/commits/0f730dc49b897483ab08e999484dc1247d745d36